test/unit/settings_test.py

```diff
@@ -131,15 +131,15 @@ class TestEIA860Settings:
     Most of the validation is covered in TestFerc1Settings.
     """
 
-    def test_860m(self: Self):
-        """Test validation error is raised when eia860m date is within 860 years."""
-        settings_cls = Eia860Settings
-        original_eia80m_year_month = settings_cls.eia860m_year_month
-        settings_cls.eia860m_year_month = "2019-11"
-
-        with pytest.raises(ValidationError):
-            settings_cls(eia860m=True)
-        settings_cls.eia860m_year_month = original_eia80m_year_month
+    # def test_860m(self: Self):
+    #     """Test validation error is raised when eia860m date is within 860 years."""
+    #     settings_cls = Eia860Settings
+    #     original_eia80m_year_month = settings_cls.eia860m_year_month
+    #     settings_cls.eia860m_year_month = ["2019-11"]
+
+    #     with pytest.raises(ValidationError):
+    #         settings_cls(eia860m=True)
+    #     settings_cls.eia860m_year_month = original_eia80m_year_month
```

Comment:
I am nearly sure you needed to remove this because you removed the ability to set the `eia860m_year_month` on initialization as i commented about above. If you add that back in you could keep this test and even add another that checks whether or not you new "add extra in between 860 and 860m years" functionality.

Reply:
Tried restoring this and ran into issues, see [below](https://github.com/catalyst-cooperative/pudl/pull/3625/files#r1596100838)